homeassistant/components/media_player/__init__.py

```diff
@@ -595,7 +595,7 @@ def entity_picture(self):
     @property
     def state_attributes(self):
         """Return the state attributes."""
-        if self.state == STATE_OFF:
+        if self.state in [STATE_OFF, STATE_UNKNOWN]:
```

Comment:
This is changing the behavior of *all* media players in unknown state. We should still try to populate the attributes if a player is in unknown state.